onnxruntime/contrib_ops/cpu/utils/dump_tensor.cc

```diff
@@ -246,21 +248,39 @@ void CpuTensorConsoleDumper::Print(const char* name, const std::string& value, b
   }
 }
 
-void CpuTensorConsoleDumper::Print(const char* name, const int32_t* tensor, gsl::span<const int64_t>& dims) const {
-  PrintTensorByDims<CpuTensorConsoleDumper, int32_t>(this, name, tensor, dims);
-}
-
-void CpuTensorConsoleDumper::Print(const char* name, const int64_t* tensor, gsl::span<const int64_t>& dims) const {
-  PrintTensorByDims<CpuTensorConsoleDumper, int64_t>(this, name, tensor, dims);
+void CpuTensorConsoleDumper::Print(const char* name, const size_t* tensor, int dim0, int dim1) const {
```

Comment:
When are we going to use this function?

Reply:
It's from existing code. It is not used and we can remove it later.